shared/settings/advanced/index.tsx

```diff
@@ -97,7 +97,7 @@ const Advanced = (props: Props) => {
             />
           </Kb.Box>
         )}
-        <ProxySettings {...props} />
+        {/* <ProxySettings {...props} /> */}
```

Comment:
may be good to add a TODO note here so people know why it's commented, but not a big deal since I'm assuming you'll be uncommenting this tomorrow.

Reply:
Done.